src/Util.ts

```diff
@@ -16,6 +18,17 @@ const parseErrorFromFailedResponse = (failureResponse: Response) =>
         .errorMap(() => new TspServiceException(TenantSecurityErrorCode.UNKNOWN_ERROR, "Unknown response from Tenant Security Proxy", failureResponse.status))
         .flatMap((errorResp: ApiErrorResponse) => Future.reject(TenantSecurityExceptionUtils.from(errorResp.code, errorResp.message, failureResponse.status)));
 
+const httpAgent = new http.Agent({keepAlive: true});
+const httpsAgent = new https.Agent({keepAlive: true});
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const agentSelector = function (_parsedURL: any) {
+    if (_parsedURL.protocol == "http:") {
```

Comment:
Why the leading `_`?

Reply:
Not sure, probably some convention I don't know. I think it works either way, but this is how I copied it.